README.md

```diff
@@ -30,7 +30,7 @@ For this repository, I've created html templates and styling files in order to b
 [Online Version Link](<https://naji-97.github.io/Portfolio-/)
 
 ## Live Demo
-[Live Demo Link](<https://github.com/naji-97/Portfolio-/assest/img/Screenshot> 2022-12-02 111444.png/)
+[Live Demo Link](<https://www.figma.com/file/l7SqJ3ZfkAKih9sFxvWSR4/Microverse-Student-Project-1?node-id=0%3A1&t=t79ZHc8kYVy3uIUT-0)
```

Comment:
- [ ] **[OPTIONAL]** KIndly replace this link with the [online version link,](https://naji-97.github.io/Portfolio-/) it is confusing to the user and the code reviewer when using the live demo link additionally the [live demo link](https://www.figma.com/file/l7SqJ3ZfkAKih9sFxvWSR4/Microverse-Student-Project-1?node-id=0%3A1&t=t79ZHc8kYVy3uIUT-0) returns the 404 error.

Reply:
Thanks